activate_wecsim.m

```diff
@@ -0,0 +1,8 @@
+% Root directory of this running .m file
+projectRootDir = pwd;
+% Add project directories to path
+addpath(fullfile(projectRootDir,genpath('source')),'-end');
+addpath(fullfile(projectRootDir,'work'),'-end');
```

Comment:
Is it possible to add a line of code to check if a 'work` folder already exists? Thus, if it gets deleted accidentally or if the WEC-Sim development team decides to not include a `work` folder in the Master code this will be generated.

Reply:
@nathanmtom  I resolved this by the most recent commit. Now the script checks if the work folder exists, and if it does not, the activate_wecsim.m creates it.